libraries/webInterface/gui/index.js

```diff
@@ -2150,4 +2162,90 @@ function addZipDownload(button, frameName) {
     });
 }
 
+function voronoiTarget(canvas, callback) {
```

Comment:
This part works great, I'm glad the pattern generator was able to be incorporated like this.